frontend/src/components/UserQuizzes/UserQuizzes.tsx

```diff
@@ -1,55 +1,47 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
 import Button from '@mui/material/Button';
 import Grid from '@mui/material/Grid';
 import { Quiz } from '../AdminQuiz';
 import QuizCard from './QuizCard';
 
-export const SampleQuizzes: Quiz[] = [
-  {
-    id: 1,
-    title: 'lorem',
-    description: 'ipsum',
-  },
-  {
-    id: 2,
-    title: 'dolor',
-    description: 'est',
-  },
-  {
-    id: 3,
-    title: 'kingkong',
-    description: 'godzilla',
-  },
-  {
-    id: 3,
-    title: 'hot',
-    description: 'dog',
-  },
-];
+import { QuizzesData, fetchQuizzes } from '../../actions';
+import { StoreState } from '../../reducers';
 
-interface Props {}
+interface Props {
+  quizzesData: QuizzesData;
+  fetchQuizzes: Function;
+}
+
+const UserQuizzes = ({ quizzesData, fetchQuizzes }: Props) => {
+  useEffect(() => {
+    fetchQuizzes();
+    return () => {};
+  }, []);
 
-const UserQuizzes = (props: Props) => {
   return (
     <div className="container mx-auto px-24 py-8">
       <h1 className="font-semibold text-2xl my-4">Quizzes</h1>
 
       <Grid container spacing={5}>
-        <Grid item>
-          <QuizCard quiz={SampleQuizzes[0]} />
-        </Grid>
-        <Grid item>
-          <QuizCard quiz={SampleQuizzes[1]} />
-        </Grid>
-        <Grid item>
-          <QuizCard quiz={SampleQuizzes[2]} />
-        </Grid>
-        <Grid item>
-          <QuizCard quiz={SampleQuizzes[3]} />
-        </Grid>
+        <>
+          {quizzesData.data?.map((quiz) => {
+            return (
+              <Grid item>
+                <QuizCard quiz={quiz} />
+              </Grid>
+            );
+          })}
+        </>
```

Comment:
Since it is wrapped by `<Grid/>` we don't need the `<></>` fragment
```suggestion
          {quizzesData.data?.map((quiz) => {
            return (
              <Grid item>
                <QuizCard quiz={quiz} />
              </Grid>
            );
          })}
```

Reply:
done